src/components/MenuItem.tsx

```diff
@@ -473,216 +472,226 @@ function MenuItem(
+                                                    fill={StyleUtils.getIconFillColor(getButtonState(focused || isHovered, pressed, success, disabled, interactive))}
                                                 />
-                                            )}
-                                        </View>
-                                    )}
-                                    {!!brickRoadIndicator && (
-                                        <View style={[styles.alignItemsCenter, styles.justifyContentCenter, styles.ml1, styles.mr2]}>
-                                            <Icon
-                                                src={Expensicons.DotIndicator}
-                                                fill={brickRoadIndicator === 'error' ? theme.danger : theme.success}
-                                            />
-                                        </View>
-                                    )}
-                                    {!title && !!rightLabel && (
-                                        <View style={styles.justifyContentCenter}>
-                                            <Text style={styles.rightLabelMenuItem}>{rightLabel}</Text>
-                                        </View>
-                                    )}
-                                    {shouldShowRightIcon && (
-                                        <View style={[styles.popoverMenuIcon, styles.pointerEventsAuto, disabled && !shouldUseDefaultCursorWhenDisabled && styles.cursorDisabled]}>
-                                            <Icon
-                                                src={iconRight}
-                                                fill={StyleUtils.getIconFillColor(getButtonState(focused || isHovered, pressed, success, disabled, interactive))}
-                                            />
-                                        </View>
-                                    )}
-                                    {shouldShowRightComponent && rightComponent}
-                                    {shouldShowSelectedState && <SelectCircle isChecked={isSelected} />}
+                                            </View>
+                                        )}
+                                        {shouldShowRightComponent && rightComponent}
+                                        {shouldShowSelectedState && <SelectCircle isChecked={isSelected} />}
+                                    </View>
                                 </View>
                                 {!!errorText && (
                                     <FormHelpMessage
                                         isError
                                         shouldShowRedDotIndicator={false}
                                         message={errorText}
-                                        style={styles.menuItemError}
+                                        style={[styles.menuItemError, styles.mt1]}
```

Comment:
```suggestion
                                        style={styles.menuItemError}
```
`menuItemError` already have marginTop 4px